src/main/java/com/ldtteam/structurize/storage/ClientBlueprintFutureProcessor.java

```diff
@@ -0,0 +1,47 @@
+package com.ldtteam.structurize.storage;
+
+import com.ldtteam.structurize.blueprints.v1.Blueprint;
+import net.minecraftforge.event.TickEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.function.Consumer;
+
+/**
+ * Waits for blueprint futures to finish loading and then processes them.
+ */
+public class ClientBlueprintFutureProcessor
+{
+    /**
+     * Queue for processing.
+     */
+    public static final Queue<ProcessingData> consumerQueue = new LinkedList<>();
```

Comment:
Don't expose fields, expose their use case in a method, so don't just add a getter.